react/week1/todo-list/src/App.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+const toDos = [
+  {
+    description: "Get out of bed",
+    deadline: "Wed Sep 13 2017"
+  },
+  {
+    description: "Brush teeth",
+    deadline: "Thu Sep 14 2017"
+  },
+  {
+    description: "Eat breakfast",
+    deadline: "Fri Sep 15 2017"
+  }
+
+]
+
+function Header () {
+  return (
+    <h1>Todo List</h1>
+  )
+}
+
+function ListItems (props) {
```

Comment:
maybe think about the naming here. should it maybe be just `ListItem`?

When passing props, you can actually use a very cool javascript feature called destructuring. For example

    const ListItems = ({description, deadline}) => { }

Then you can access the variables without using `props.`